### PR TITLE
Task overview -page

### PR DESCRIPTION
**Pull request by IidaKainu**
Adds the task overview -page with the first section 'task overview'.

### CODE REVIEW

File: frontend/src/components/TaskOverview.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      <div className={classes(css.metrics)}>
+        {metrics.map(({ label, metricsValue, children }) => (
+          <MetricsSummary key={label} label={t(label)} value={metricsValue}>
+            {children}
+          </MetricsSummary>
+        ))}
+      </div>
+      <div className={classes(css.data)}>
+        {taskData.map((column, idx) => (
+          <div className={classes(css.dataColumn)} key={idx}>
```
Comment (PatricKangasmaki) [inline frontend/src/components/TaskOverview.tsx:67]: One thing that I noticed: This file gives eslint complaints. I think this warning could be suppressed (as it's valid to use the index as a key here so there's no need to refactor it) so the editor complaints don't cause confusion later on
```suggestion
          // eslint-disable-next-line react/no-array-index-key
          <div className={classes(css.dataColumn)} key={idx}>
```

**Review by PatricKangasmaki — APPROVED** [review]
Good quality code, nice work! 👍

File: frontend/src/pages/TaskOverviewPage.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            to={`${paths.roadmapHome}/${roadmapId}${paths.roadmapRelative.taskList}`}
+          >
+            <ArrowBackIcon className={classes(css.arrowIcon)} />
+          </Link>
+          <Trans i18nKey="Task overview" />
+          <div className={classes(css.taskName)}>{task?.name}</div>
+          <div className={classes(css.buttons)}>
+            {siblingTasks.map(({ id, type }) => (
+              <IconButton
+                className={classes({ [css.disabled]: !id })}
```
Comment (juhoeerola) [inline frontend/src/pages/TaskOverviewPage.tsx:48]: The `IconButton` is missing a `key`

**Review by juhoeerola — COMMENTED** [review]
Noticed one missing key, otherwise looks great 👍

**Comment by IidaKainu** [comment]
Thanks for the suggestions, I added them in 🙏

**Pull request merged**